src/main/Welcome.java

```diff
@@ -0,0 +1,549 @@
+		directionsLabel.setBounds(0, 380, 1136, 73);
+		welcomePanel.add(directionsLabel);
+
+		JLabel logoLabel = new JLabel("Logo");
+		logoLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		logoLabel.setBounds(0, 20, 211, 35);
+		taskbarPanel.add(logoLabel);
+
+		JButton accountButton = new JButton("My Account");
+		accountButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				introPanel.setVisible(false);
+				accountPanel.setVisible(true);
+			}
+		});
+		accountButton.setBounds(25, 180, 160, 23);
+		taskbarPanel.add(accountButton);
+
+		JButton quizButton = new JButton("Take Quiz");
+		quizButton.setBounds(25, 240, 160, 23);
+		taskbarPanel.add(quizButton);
+
+		JButton resultsButton = new JButton("View Results");
+		resultsButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				accountPanel.setVisible(false);
+				resultsPanel.setVisible(true);
+			}
+		});
+		resultsButton.setBounds(25, 300, 160, 23);
+		taskbarPanel.add(resultsButton);
+
+		JButton helpButton2 = new JButton("Help");
+		helpButton2.setBounds(45, 500, 120, 23);
+		taskbarPanel.add(helpButton2);
+
+		JButton signOutButton = new JButton("Sign Out");
+		signOutButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				dashboardPanel.setVisible(false);
+				welcomePanel.setVisible(true);
+			}
+		});
+		signOutButton.setBounds(45, 550, 120, 23);
+		taskbarPanel.add(signOutButton);
+
+		JButton uniInfoButton = new JButton("University Information");
+		uniInfoButton.setBounds(25, 116, 160, 23);
+		taskbarPanel.add(uniInfoButton);
+
+		JLabel titleLabel2 = new JLabel("App Name");
+		titleLabel2.setFont(new Font("Tahoma", Font.PLAIN, 36));
+		titleLabel2.setHorizontalAlignment(SwingConstants.CENTER);
+		titleLabel2.setBounds(0, 33, 926, 90);
+		introPanel.add(titleLabel2);
+
+		JLabel quoteLabel = new JLabel("Quote");
+		quoteLabel.setFont(new Font("Tahoma", Font.PLAIN, 16));
+		quoteLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		quoteLabel.setBounds(0, 158, 926, 76);
+		introPanel.add(quoteLabel);
+
+		JLabel creatorLabel = new JLabel("Created By:");
+		creatorLabel.setFont(new Font("Tahoma", Font.PLAIN, 20));
+		creatorLabel.setBounds(20, 550, 896, 59);
+		introPanel.add(creatorLabel);
+
+		JLabel personsLabel = new JLabel("Person's Account");
+		personsLabel.setFont(new Font("Tahoma", Font.PLAIN, 28));
+		personsLabel.setHorizontalAlignment(SwingConstants.CENTER);
+		personsLabel.setBounds(50, 30, 280, 65);
+		accountPanel.add(personsLabel);
+
+		JLabel courseCaptionLabel = new JLabel("Courses and Grades");
+		courseCaptionLabel.setFont(new Font("Tahoma", Font.PLAIN, 20));
+		courseCaptionLabel.setHorizontalAlignment(SwingConstants.LEFT);
+		courseCaptionLabel.setBounds(30, 130, 200, 75);
+		accountPanel.add(courseCaptionLabel);
+
+		JLabel course1Label = new JLabel("Course 1 + Grade");
```

Comment:
I understand that windows builder generated all this, but when we add the logic, it'll be helpful if the labels are in an array (since I would imagine that the marks would probably be in an array for the user object)